src/components/bucket/GroupDataList.tsx

```diff
@@ -66,7 +67,7 @@ export function DataList(props) {
 }
 
 export function ListActions({ bid, session, bucket }) {
-  if (session.busy || bucket.busy) {
+  if (session.busy || bucket.busy || !canEditBucket(session, bucket)) {
```

Comment:
This should probably be `!canCreateGroup()` no?

Reply:
Wow I totally missed that function and thought it was lumped in with bucket edit. Thank you for catching that.